tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+jobs:
```

Comment:
This file contains quite a bit of duplicate "code" with the other files.
We could either solve that by simplifying this file, OR by moving the "compile" stage into a separate file and sharing it across all other files.

I would first propose to try simplifying it.

Reply:
Yes. I would try to simplify the file to avoid duplicate code